src/api/app/lib/backend/test/tasks.rb

```diff
@@ -49,6 +25,16 @@ def run_admin(args)
         end
         ret
       end
+
+      private
+
+      def run_backend_server(name:, command:)
```

Comment:
this name is highly confusing, could you use something like "single_scheduler_evaluation_run" or something like that?

Reply:
hm, or something like "execute_backend_daemon".

but "backend_server" are many processes for me, not a single temporary execution.

Reply:
"single_scheduler_evaluation_run" could be more confusing. This method not only runs a scheduler. I'm open to more ideas.

Reply:
I renamed the method to `execute_backend_daemon`.